api/proxy.config.ts

```diff
@@ -1,113 +1,115 @@
-import * as bodyParser from 'body-parser'
-import {Express} from 'express'
-import * as amendedJurisdictions from './amendedJurisdictions'
-import {getConfigValue} from './configuration'
-import {
-    SERVICES_CCD_COMPONENT_API_PATH,
-    SERVICES_DOCUMENTS_API_PATH, SERVICES_EM_ANNO_API_URL, SERVICES_EM_HRS_API_PATH,
-    SERVICES_ICP_API_URL, SERVICES_MARKUP_API_URL, SERVICES_PAYMENTS_URL
-} from './configuration/references'
-import {applyProxy} from './lib/middleware/proxy'
-import * as searchCases from './searchCases'
+import * as bodyParser from 'body-parser';
+import { Express } from 'express';
+
+import { checkCachedJurisdictions, getJurisdictions } from './amendedJurisdictions';
+import { getConfigValue, proxiedReferences } from './configuration';
+import { applyProxy } from './lib/middleware/proxy';
+import { handleElasticSearchResponse, modifyRequest } from './searchCases';
 
 export const initProxy = (app: Express) => {
-    applyProxy(app, {
-        rewrite: true,
-        rewriteUrl: '/activity',
-        source: [
-            '/activity',
-        ],
-        target: getConfigValue(SERVICES_CCD_COMPONENT_API_PATH),
-    })
+  applyProxy(app, {
+    rewrite: true,
+    rewriteUrl: '/socket.io',
```

Comment:
(Nitpick)
Does the url have to be socket.io? Bit revealing.

Reply:
It _doesn't_ have to be, but that's the default URL and I was having some difficulty with the socket.io configuration to get it to point to anything else relative. We can't use /activity either because it'll clash with the other proxy directly beneath. I'll have another look to see what we can do.

Reply:
Had a good go at this but it's being a nuisance. I think I'll leave it for the time being seeing as it's currently working and you had it down as a _nitpick_ anyway. 😁